src/test/java/uk/gov/hmcts/reform/bulkscan/orchestrator/model/ccd/mappers/SupplementaryEvidenceMapperTest.java

```diff
@@ -32,7 +33,7 @@ public void from_envelope_maps_all_fields_correctly() {
 
         assertThat(scannedDocument.controlNumber).isEqualTo(envelopeDocument.controlNumber);
         assertThat(scannedDocument.fileName).isEqualTo(envelopeDocument.fileName);
-        assertThat(scannedDocument.type).isEqualTo(envelopeDocument.type);
```

Comment:
Should we keep the assertion for `type`?

Reply:
I think we should, to check if subtype value is copied to type